client/js/main.js

```diff
@@ -137,7 +139,11 @@ function getDocsFromBertha (docs, republish = false) {
 			}
 
 			// Set the options globally
-			parseOptions(config);
+			const sheetTitle = parseOptions(config).title || undefined;
```

Comment:
Parse options does override values, but we don't populate the title from this variable. For every sheet that gets loaded into our app, we run it's config params through this function and set this variable each time. If we get a title value, then we push that value the the global array of sheetTitles (terrible name, I know) and when all of our sheets have been processed, we populate the titles value from all of the values stored in that array joined by ' & '. So, one should never override the others, there may be duplicates if multiple sheets have the same title (which we can filter), but I don't see how what you've described can occur.